UI/ui.py

```diff
@@ -5,7 +5,7 @@
 from UI.passFail import passFail
 
 
-def mainMenu():
+def mainMenu(**kwargs):
```

Comment:
Not quite - we dont need this here

Reply:
We want this [here](https://github.com/TriHardStudios/101GradingScript/blob/f95c04980a77e44863516bb2d60b152a09ae847f/UI/standardGrading.py#L7)